docpages/example_code/embeds.cpp

```diff
@@ -2,7 +2,9 @@
 
 int main() {
 	/* Setup the bot */
-	dpp::cluster bot("token", dpp::i_default_intents | dpp::i_message_content);
+	dpp::cluster bot("token", dpp::i_default_intents);
```

Comment:
Just change this to `dpp::cluster bot("token");`. It's default intents by default, not needed

Reply:
Done.